config/test/test-setup.js

```diff
@@ -5,11 +5,15 @@
  */
 import MutationObserver from "@sheerun/mutationobserver-shim";
 import {configure} from "@testing-library/dom"; // eslint-disable-line testing-library/no-dom-import, prettier/prettier
+import {StyleSheetTestUtils} from "aphrodite";
 import Enzyme from "enzyme"; // eslint-disable-line no-restricted-imports
 import React16EnzymeAdapter from "enzyme-adapter-react-16"; // eslint-disable-line no-restricted-imports
 import jestSerializerHtml from "jest-serializer-html";
 import {addSerializer} from "jest-specific-snapshot";
 
+// Prevent aphrodite from trying to inject styles into the CSSOM
+StyleSheetTestUtils.suppressStyleInjection();
```

Comment:
I saw some errors involving Aphrodite so I added this line which was missing from our test setup.